doc/ale-mercury.txt

```diff
@@ -0,0 +1,28 @@
+===============================================================================
+ALE Mercury Integration
+*ale-mercury-options*
+
+
+===============================================================================
+mmc                                                           *ale-mercury-mmc*
+
+
+g:ale_mercury_mmc_executable
```

Comment:
Write `g:...   *g:...*`, with the first documentation marker on the same line as the variable name, if it all fits within the maximum column size. Have a look at the other files for examples.